README.md

```diff
@@ -1,14 +1,164 @@
+  common integration tests responsible for running unit tests and linters
+  (including coding style checks), and computing the coverage. In the future
+  we are planning to test the integration of the rust-vmm components by
+  providing a reference VMM implementation.
+- **Clean interface**. With shared crates you also have to think about an
+  interface flexible enough to be used by multiple VMMs. We have multiple
+  rounds of design reviews up to the point we are confident the interface is
+  clean and reusable by other projects.
+
+## Status of rust-vmm Components
+
+Keep in mind that this status might not always be up to date. For the latest
+status please check individual repositories.
+
+### Coming Soon
+
+These are the empty repositories that have PRs waiting to be merged.
+
+- [linux-loader](https://github.com/rust-vmm/linux-loader): parser and loader
+  for vmlinux and bzImage images as well as some other helpers for kernel
+  commandline.
+- [virtio-bindings](https://github.com/rust-vmm/virtio-bindings): Rust FFI
+  bindings to virtio kernel headers generated using
+  [bindgen](https://crates.io/crates/bindgen).
+- [vm-virtio](https://github.com/rust-vmm/vm-virtio/): virtio device trait and
+  implementation for virtio primitives such as virtqueues and descriptor chain.
+- [vmm-sys-utils](https://github.com/rust-vmm/vmm-sys-util/): a collection of
+  modules providing helpers and utilities for building VMMs and Hypervisors.
+- [vmm-vcpu](https://github.com/rust-vmm/vmm-vcpu/): a hypervisor-agnostic
+  abstraction for Virtual CPUs (vCPUs).
+
+### Existing Crates
+
+These are crates for which we have most of the code out there, but
+there might be some missing requirements for publishing them on crates.io.
+For the requirement list refer to the
+[Publishing on crates.io - Requirements list](#publishing-on-crates.io---requirements-list)
+section.
+
+- [vm-memory](https://github.com/rust-vmm/vm-memory): abstractions over a
+  Virtual Machine memory.
+
+#### Components Published on [crates.io](crates.io)
+
+These are production-ready components that can be pulled as dependencies
+directly from crates.io.
+
+- [kvm-bindings](https://crates.io/crates/kvm-bindings): Rust FFI bindings
+  to KVM generated using [bindgen](https://crates.io/crates/bindgen).
+- [kvm-ioctls](https://crates.io/crates/kvm-ioctls): Safe wrappers over the
+  KVM API.
+
+## Development
+
+### Adding a New Virtualization Component
+
+If you have an idea about a component that could be a great fit for rust-vmm,
+please open a
+[review request issue](https://github.com/rust-vmm/community/issues/new?assignees=&labels=&template=new-crate-request.md&title=Crate+Addition+Request).
+The issue must provide the explanations and justifications behind the need for
+a new component, highlighting the purpose of this new repository, how will it
+be used by other VMMs or Hypervisors, and other information that you find
```

Comment:
@aghecenco Let me first clarify the vmm vs hypervisor thing first. Some of the rust-vmm components can be used to create hypervisors as well, there's nobody saying you can only create VMMs. So that's why throughout the doc you'll see VMMs or Hypervisors. I am not trying to say you can use VMM and Hypervisors interchangeably, I hope that's not how it reads like.

@raduweiss do you mean I should use Virtual Machine Managers instead of VMMs?

Reply:
No no, that was one of my two sarcastic comments.